ansible_collections/arista/avd/docs/porting-guides/4.x.x.md

```diff
@@ -83,6 +83,33 @@ hardware_counters:
       direction: in
 ```
 
+### New model for `ip_name_servers`
+
+The `name_server` key has been deprecated in favor of `ip_name_servers`, more
+aligned with the EOS cli.
+
+TODO: add link to the full data model
+
+Example with old data model:
+
+```yaml
+name_server:
+  source:
+    vrf: MGMT
+  nodes:
+    - 8.8.8.8
+```
+
+Same example with new data model:
+
+```yaml
+ip_name_servers:
```

Comment:
We should not just rename the key. The idea should be to create a new data model like:
```yaml
ip_name_servers:
  - ip_address: <str>
    vrf: <vrf>
```
To align closer to the CLI. (Please verify)

Reply:
```
device(config)#show cli commands | grep "ip name-server"
[no|default] ip name-server [ ( VRF_VRF_KW ( VRF_VRF_DYNAMIC ) ) ] { V4 | V6 } [ priority PRIORITY ]
```

Makes sense.. the issue could have been more explicit..